rakelib/extension/cross_rubies.rb

```diff
@@ -0,0 +1,134 @@
+# frozen_string_literal: true
+
+CrossRuby = Struct.new(:version, :platform) do
```

Comment:
FWIW, I’ve tried to [package some of this data in rb-sys](https://github.com/oxidize-rb/actions/blob/21b8acf931df3286b9cc4d3a7366b744f9bb4791/setup-ruby-and-rust/action.yml#L50), and would love to add support for anything else that’s useful if it’s missing anything!

Reply:
Hell yeah, thanks!

Reply:
That link goes to the wrong project, I think--is the rb-sys logic in `RbSys::ToolchainInfo` ? I'm not seeing an easy way to just do like `if windows?` using pure rb-sys.

Reply:
Good point, seems useful to have. I'd be happy to add that.